code/frameworks/svelte-vite/package.json

```diff
@@ -51,7 +51,7 @@
     "prep": "../../../scripts/prepare/bundle.ts"
   },
   "dependencies": {
-    "@storybook/addon-svelte-csf": "^2.0.0",
+    "@storybook/addon-svelte-csf": "^3.0.0-next.0",
```

Comment:
Why is this both a dependency and a peer dependency? I think it shouldn't be a dependency @JReinhold @IanVS

Reply:
In fact it should be removed entirely, from both peer _and_ dependencies.  We included it previously in peer dependencies because we were using it in the `csf-plugin`.  But now we don't need to with the latest version of the addon.

@JReinhold there are also some type definition files that can be removed now too, along with `plugins/csf-plugin.ts`